pkgs/by-name/de/deark/package.nix

```
@@ -0,0 +1,68 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  help2man,
+}:
+stdenv.mkDerivation rec {
+  pname = "deark";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "jsummers";
+    repo = "deark";
+    tag = "v${version}";
+    hash = "sha256-dyX41gWZnZ/07Vyxo1x4Y8neGHS5ev+YyBJ0cUH+gKY=";
+  };
+
+  nativeBuildInputs = [ help2man ];
+  postBuild = ''
+    make man
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 deark $out/bin/deark
+    install -Dm644 deark.1 $out/share/man/man1/deark.1
```

```suggestion
  nativeBuildInputs = [
    help2man
    installShellFiles
  ];
  postBuild = ''
    make man
  '';

  installPhase = ''
    runHook preInstall

    install -Dm755 deark $out/bin/deark
    installManPage deark.1
```

Sorry, didn't see this one

---

No worries, thanks for the feedback; pushing now